### ISSUE TITLE
Half-Orc Racial Traits

### ISSUE THREAD

**Issue by Lamorak11** [mlenser/kryx-rpg-issues#335]
The Savage Attacker trait is too focused on melee combat, and pushes players towards only playing melee Half-Orcs. 

Half-Orcs need a replacement trait that is more suited to all class types, while maintaining the flavour of an aggressive orcish nature.

**Comment by Lamorak11** [comment]
Perhaps something like this could work:

**Bloody-minded**. You can call upon the orcish fury that waits just beneath the surface to push you through when you need it most.

As a reaction, which you can use when you roll a saving throw, you can reroll the saving throw using your proficiency bonus plus your strength modifier instead of the usual saving throw, taking the second result.

Once you use this feature, you must finish a long rest before you can use it again.

**Comment by mlenser** [comment]
I think Relentless Endurance covers the "tough" side of Orcs:
> **_Relentless Endurance._** When you are reduced to 0 hit points but not killed outright, you can drop to 1 hit point instead. Once you use this feature, you must finish a long rest before you can use it again.

I think I agree with the discussion on discord: I'd like to see something offensive.

**Comment by Lamorak11** [comment]
Kryx, I really liked your comment about "Blood Fury" from WoW orcs:

“Orcs can fly into a rage that increases their attack power or intellect (based on class) for a short time. “
 
Something like that would be great, although it might be difficult to reach mechanics that fit it in d&d. A mini-rage that grants a meaningful bonus to a character regardless of their class.

**Comment by Lamorak11** [comment]
Perhaps something like this could be workable?

**Blood Fury**. When you make an attack roll or cast a damage-dealing spell, you can unleash the orcish fury that seethes through your blood, causing it to deal 1d4 extra damage. The extra damage increases to 1d6 at 5th level, 1d8 at 9th level, 1d10 at 13th level and 1d12 at 17th level.
&emsp;&emsp;Once you use this trait, you can’t use it again until you finish a short or long rest.

**Comment by mlenser** [comment]
Comparison to Fury of the small:

### Level 1
- Blood Fury: ~2.5 damage 1/short rest
- Fury of the small: 2 damage 1/short rest

### Level 5
- Blood Fury: ~3.5 damage 1/short rest
- Fury of the small: 4 damage 1/short rest

### Level 9
- Blood Fury: ~4.5 damage 1/short rest
- Fury of the small: 6 damage 2/short rest

### Level 13
- Blood Fury: ~5.5 damage 1/short rest
- Fury of the small: 8 damage 2/short rest

### Level 17
- Blood Fury: ~6.5 damage 1/short rest
- Fury of the small: 10 damage 2/short rest

_____________________

So pretty similar, though I'd say we'd need to change it to 2/short rest at 9th level.

It feels slightly lackluster though.

**Comment by mlenser** [comment]
Maybe something like
> **_Blood Fury._** As a bonus action, you can unleash the orcish fury seething through your blood. Once on each of your turns, you can add your proficiency bonus to the damage of a melee weapon or a spell. Your fury lasts for 1 minute. It ends early if you are knocked unconscious.
> 
> Once you use this trait, you can’t use it again until you finish a long rest.
> 
> At 9th level you regain the ability to use this trait when you finish a short or long rest.

**Comment by itamarcu** [comment]
Why not ranged weapons or unarmed attacks?

**Comment by mlenser** [comment]
The same reason that a Berserker's rage does not apply to ranged weapons. Thrown weapons count, but not things like blowguns or bows.

I can reword it to `To the damage of a weapon attack using Strength or a spell`

**Comment by mlenser** [comment]
Orc Savage Attacks replaced with Blood Fury

![image](https://user-images.githubusercontent.com/235857/84410053-27794500-ac0e-11ea-9ac1-58e4531351fa.png)

### Calculations:
Combats last ~5 rounds, so lets use that as a duration

Level 1:
- Blood Fury: 2x5 = ~10 damage over 5 rounds 1/short rest
- Goliath's Stone's Endurance reduces damage by 6.5+level+con = 10.5 1/short rest.

Level 9:
- Blood Fury: 4x5 = 20 over 5 rounds 1/short rest
- Stone's Endurance 6.5+9+4 = 19.5 1/short rest
